web_client/utils/analysis.jsx

```diff
@@ -141,14 +141,15 @@ export const runTask = (taskName, params, options = {}) => {
       .reduce(objectReduce, {})
   );
 
-  return (
-    rest({ path, type: 'POST', data })
-      .then(({
-        response: {
-          job: { _id: id }
-        }
-      }) => pollJob(id, title, taskName, pollInterval, 0, maxPolls))
-  );
+  return rest({ path, type: 'POST', data })
+    .then(({ response }) => response)
```

Comment:
Just an opinion, but I think this would be clearer as

```javascript
x => x.response
```

As written, it repeats `response` and on a very quick glance, looks much like the identity function. The style I suggested is a standard idiom for a "plucker" function that maps an object to one of its values.